x-pack/solutions/security/plugins/lists/server/services/exception_lists/build_exception_filter.ts

```diff
@@ -329,6 +340,16 @@ export const buildExceptionFilter = async <
     listClient,
   });
 
+  agent.setCustomContext({
+    exceptions: {
+      [SECURITY_NUM_LARGE_LIST_EXCEPTIONS]:
+        unprocessableValueListExceptions.length + unprocessableExceptionItems.length,
```

Comment:
Assuming that these context keys are relatively inexpensive: it seems more robust to capture all of these "raw" values rather than just the derived, lossy values, but perhaps we'll never need to care e.g. how many `unprocessableExceptionItems` we have?

Reply:
I was thinking it'd be more convenient to have the total large, small, and regular numbers since that provides extra context about the exception filter that we're including in the generated rule query. The raw values could provide details about how we're generating the exception filter, i.e. the `unprocessableExceptionItems` require an extra value list client call to fetch the list and figure out how many dash notation items there are for ip_range lists - but APM will already instrument those calls individually so that info isn't completely lost.

I propose that we start with capturing the higher level numbers and add more detailed context later on.

Reply:
Makes sense to me; thanks!